src/interfaces/srv/VideoOut.srv

```diff
@@ -0,0 +1,11 @@
+# VideoOut.srv
+
+int32 height
```

Comment:
Maybe im not understanding the two messages, is the VideoOut a combined stream of x amount of VideoSources? So a combined video of x and y dimensions, and then the sources array has the component videos with its original sizes. Thats at least how I'm understanding how the node and pipeline work just want to make sure

Reply:
Yes that is correct. The videoOut service describes the height and width of entire canvas (multiple sources) and has a list of videoSources which describe the location and size of each specific camera stream on the canvas.